DeUrgenta/Features/CreateGroup/View/ContactRow.swift

```diff
@@ -0,0 +1,67 @@
+import MessageUI
+import SwiftUI
+
+protocol ContactRowDelegate {
+    func contactRowDidTapAddButton(from view: ContactRow, withItem contact: ContactInfo)
+}
+
+struct ContactRow: View {
+    @ObservedObject var viewModel: CreateGroupViewModel
+    @State var showingAlert = false
+
+    var contact: ContactInfo
+    var delegate: ContactRowDelegate
+
+    var body: some View {
+        VStack {
+            HStack {
+                Image("logo")
+                    .resizable()
+                    .frame(width: 32, height: 32)
+
+                VStack(alignment: .leading) {
+                    Text("\(contact.firstName) \(contact.lastName)")
+                        .font(.custom("IBMPlexSans-Regular", size: 15))
+                        .foregroundColor(.darkText)
+
+                    Text("\(contact.phoneNumber?.stringValue ?? "")")
+                        .font(.custom("IBMPlexSans-Regular", size: 15))
+                        .foregroundColor(.gray)
+                }
+                .lineSpacing(2)
+
+                Spacer()
+
+                Button(action: {
+                    if viewModel.invitedContacts.contains(where: { $0.id == contact.id }) {
+                        showingAlert = true
+                    } else {
+                        delegate.contactRowDidTapAddButton(from: self, withItem: contact)
+                    }
+
+                }, label: {
+                    Text(AppStrings.ContactRow.buttonText.localized())
+                        .foregroundColor(Color.accent)
+                        .padding(.bottom, 2)
```

Comment:
* You can merge .bottom and .top padding in padding(.vertical, 2)
* You can merge .leading and .trailing padding in padding(.horizontal, 10)